src/components/Sum/Sum.jsx

```diff
@@ -1 +1,8 @@
-// export const Sum = () => ();
+export const Sum = ({
+  a = 0,
+  b = 0,
+}) => (
```

Comment:
```suggestion
export const Sum = ({ a = 0, b = 0 }) => (
```
in this case not required to make each prop on new line.